<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>검색어 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outputBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>421</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>현재 수집 개수 : 0개</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
